wp-content/themes/currentorg/partials/sidebar-wpbdp-listing-sidebar.php

```diff
@@ -1,5 +1,23 @@
 <div class="listing-sidebar">
 
+    <?php
+
+        $listing_id = $wp_query->query_vars['_wpbdp_listing'];
+        $listing_logo = wpbdp_get_form_field( 11 );
+
+        if ( is_object( $listing_logo ) && $listing_logo->value( $listing_id ) ){
+
+            printf(
+                '<div class="listing-main-image">
+                    <img src="%1$s">
+                </div>',
+                $listing_logo->value( $listing_id )
```

Comment:
I'm receiving the following error:

> Notice: Array to string conversion in /Users/blk/sites/current/wp-content/themes/currentorg/partials/sidebar-wpbdp-listing-sidebar.php on line 14

The image link is https://current.test/directory-of-services/2531317/test-company-12/Array/

Reply:
![Screen Shot 2019-07-09 at 10 57 22 ](https://user-images.githubusercontent.com/1754187/60899364-59cd3600-a238-11e9-9b86-067c36c48b5f.png)